### ISSUE TITLE
[dashboard] Dashboard doesn't load

### ISSUE THREAD

**Issue by mattearllongshot** [ray-project/ray#10094]
### What is the problem?

OS: Ubuntu 18.04.4 LTS
Ray: 0.8.6
Python: 3.8.1

After opening the dashboard URL, the page starts to load but shortly after goes white, and the following is shown in the console:
![image](https://user-images.githubusercontent.com/37295291/90118621-6c297580-dd50-11ea-81cc-362fc874681a.png)

`dashboard.err` on the head node shows a mixture of these two exceptions:

```
Traceback (most recent call last):
  File "/home/ubuntu/conda/envs/venv/lib/python3.8/site-packages/ray/dashboard/dashboard.py", line 694, in run
    reply = stub.GetNodeStats(
  File "/home/ubuntu/conda/envs/venv/lib/python3.8/site-packages/grpc/_channel.py", line 826, in __call__
    return _end_unary_response_blocking(state, call, False, None)
  File "/home/ubuntu/conda/envs/venv/lib/python3.8/site-packages/grpc/_channel.py", line 729, in _end_unary_response_blocking
    raise _InactiveRpcError(state)
grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
        status = StatusCode.UNAVAILABLE
        details = "failed to connect to all addresses"
        debug_error_string = "{"created":"@1597310306.233611716","description":"Failed to pick subchannel","file":"src/core/ext/filters/client_channel/client_channel.cc","file_line":3981,"referenced_errors":[{"created":"@1597310304.500921719","description":"failed to connect to all addresses","file":"src/core/ext/filters/client_channel/lb_policy/pick_first/pick_first.cc","file_line":394,"grpc_status":14}]}"
>
```

```
Traceback (most recent call last):
  File "/home/ubuntu/conda/envs/venv/lib/python3.8/site-packages/ray/dashboard/dashboard.py", line 694, in run
    reply = stub.GetNodeStats(
  File "/home/ubuntu/conda/envs/venv/lib/python3.8/site-packages/grpc/_channel.py", line 826, in __call__
    return _end_unary_response_blocking(state, call, False, None)
  File "/home/ubuntu/conda/envs/venv/lib/python3.8/site-packages/grpc/_channel.py", line 729, in _end_unary_response_blocking
    raise _InactiveRpcError(state)
grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
        status = StatusCode.DEADLINE_EXCEEDED
        details = "Deadline Exceeded"
        debug_error_string = "{"created":"@1597310309.640169697","description":"Deadline Exceeded","file":"src/core/ext/filters/deadline/deadline_filter.cc","file_line":69,"grpc_status":4}"
>
````

### Reproduction (REQUIRED)

Boot a cluster, spin up some workers (may or may not be necessary).  Load the dashboard URL in Firefox

- [ ] I have verified my script runs in a clean environment and reproduces the issue.
- [ ] I have verified the issue also occurs with the [latest wheels](https://docs.ray.io/en/latest/installation.html).

**Comment by mfitton** [comment]
@mattearllongshot I'm sorry that you've had this problem as well. This is unfortunately a known bug with the 0.8.6 release. The issue is fixed in our newest release 0.8.7. Please read here for more details: https://github.com/ray-project/ray/issues/9317

**Comment by mattearllongshot** [comment]
Great thanks, I'll switch to 0.8.7
